x/group/server/invariants.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/regen-network/regen-ledger/orm"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func (s serverImpl) RegisterInvariants(ir sdk.InvariantRegistry) {
+	ir.RegisterRoute(group.ModuleName, "Tally-Votes", s.tallyVotesInvariant())
```

Comment:
I would put `"Tally-Votes"` in a private const in this file

Reply:
Okay, let's do it that way